Source/Tools/LinuxEmulation/LinuxSyscalls/Utils/Threads.cpp

```diff
@@ -61,6 +82,32 @@ namespace FEX::LinuxEmulation::Threads {
     AddStackToDeadPool(Ptr);
```

Comment:
Just to make sure, is it intentional that this code path doesn't set `ReadyToBeReaped`?

Reply:
Intentional stack memory leak that is happening elsewhere and is a pre-existing condition. Subject to refactoring that has yet to occur. It's one of the reasons why I've spent months moving thread management to the frontend.